src/pages/VotePage.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import VoteCell from '../components/VoteCell';
+
+export default function VotePage() {
+  const [candidates, setCandidates] = useState([]);
+
+  async function getCandidates() {
+    // GET, Candidates
+    const response = await axios({
+      method: 'get',
+      url: 'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates',
+      responseType: 'stream',
+    });
+
+    const { status, data } = response;
+
+    switch (status) {
+      case 200:
+        // 투표순위대로 정렬
+        data.sort((a, b) => {
+          return b.voteCount - a.voteCount;
+        });
```

Comment:
화살표함수에서 바로 `return`할땐 `중괄호`와 `return`문을 **생략**할 수 있습니다~
```suggestion
        data.sort((a, b) => b.voteCount - a.voteCount);
```